HW0/P5/P5.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import math
+
+# Main
+if __name__ == '__main__':
+	x = np.linspace(1, 1000, 20)
+	y1 = [math.log(i, 2) for i in x]
+	y2 = [(i - 1) for i in x]
+	plt.plot(x, y1, '-b', label = 'Infinite number of employees')
+	plt.plot(x, y2, '--r', label = 'Lone cashier')
```

Comment:
It would be great to plot this on a log scale, since we can't really see what the parallelized plot is doing.